pyiron/vasp/outcar.py

```diff
@@ -474,7 +474,11 @@ def get_broyden_mixing_mesh(filename="OUTCAR", lines=None):
             int: Mesh size
         """
         trigger_indices, lines = _get_trigger(lines=lines, filename=filename, trigger="gives a total of ")
-        line_ngx = lines[trigger_indices[0]-2]
+        try:
+            line_ngx = lines[trigger_indices[0]-2]
+        except IndexError:
```

Comment:
I do not really like the `try except` style, because in my feeling this might surpress other errors, therefore I would prefer to check the file size and raise an error directly.

Reply:
This fix is unrelated to the CHGCAR parsing fix. This is for calculations using the line mode kpoints (for band structures like Felix does) where theese lines are missing. I think this is an appropriate way to handle this. This pull request handles bugs in the OUTCAR parser as well.

Reply:
Fixed [here](https://github.com/pyiron/pyiron/pull/185/commits/8698b6641ca5b311f288b3a886e691eb69b0225f)

Reply:
Can not we simply check the length of the `trigger_indices` ?

Reply:
Yes. But the only possible error that can be raised in this case is an `IndexError`. In 99% of the cases the `if` statement will be `True`. So using the [EAFP philosophy](https://docs.quantifiedcode.com/python-anti-patterns/readability/asking_for_permission_instead_of_forgiveness_when_working_with_files.html) a `try` and `except` statement would be better

Reply:
That is right. Personally I still prefer to have the if statement, as it is just one line of code and I can not see how it helps to clean the code by adding more lines.

Reply:
Okay, implemented [here](https://github.com/pyiron/pyiron/pull/185/commits/a8a2c86913438865132280e3d7a3ae29ba94c86b)